ggml.c

```diff
@@ -1548,15 +1550,15 @@ static void ggml_vec_dot_q8_0_q8_0(const int n, float * restrict s, const void *
 
 static const quantize_fns_t quantize_fns[GGML_TYPE_COUNT] = {
     [GGML_TYPE_Q4_0] = {
-        .dequantize_row_q         = (dequantize_row_q_t) dequantize_row_q4_0,
+        .dequantize_row_q         = (dequantize_row_q_t) ggml_vk_dequantize_row_q4_0,
         .quantize_row_q           = quantize_row_q4_0,
         .quantize_row_q_reference = (quantize_row_q_t) quantize_row_q4_0_reference,
         .quantize_row_q_dot       = quantize_row_q8_0,
         .vec_dot_q                = ggml_vec_dot_q4_0_q8_0,
         .vec_dot_type             = GGML_TYPE_Q8_0,
     },
     [GGML_TYPE_Q4_1] = {
-        .dequantize_row_q         = (dequantize_row_q_t)dequantize_row_q4_1,
+        .dequantize_row_q         = (dequantize_row_q_t) ggml_vk_dequantize_row_q4_1,
```

Comment:
Should not change these - I assume these are temporary

Reply:
Yup